idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -486,12 +491,22 @@ def _make_performance(self):
                                       initialize=160,
                                       doc='Enhancement factor')
 
-        self.yi_MEA = Var(self.flowsheet().time,
+        # self.yi_MEA = Var(self.flowsheet().time,
+        #                   self.liquid_phase.length_domain,
+        #                   domain=NonNegativeReals,
+        #                   initialize=0.5,
+        #                   units=None,
+        #                   doc='''Dimensionless concentration of MEA
+        #                             at interface ''')
+        
+        self.yi_solvent = Var(self.flowsheet().time,
                           self.liquid_phase.length_domain,
+                          solvent_comp_list,
```

Comment:
1. Why not include H2O in here as well for simplicity? It might not get used elsewhere, but most property packages will not specify this.
2. As there is an equivalent term for solutes as well, I would just make a single component for this, and have the rule decide whether to use the solvent or solute approach.
3. We should call this `conc_interface` or something similar.
4. See also the comment below about `self.equilibrium_components` - these might be the only components that need to appear here, so we might be able to use that as the indexing set here.